clustered/integration-test/build.gradle

```diff
@@ -41,7 +40,6 @@ dependencies {
   serverLibs ("org.terracotta.management.dist:management-server:$parent.managementVersion") {
     exclude group:'org.terracotta.management.dist', module:'management-common'
   }
-  serverLibs "org.terracotta.management:monitoring-service-entity:$parent.managementVersion" // test entity for monitoring service
```

Comment:
no need anymore: we cab directly use the tms entity to read the monitoring state